lib/mqtt/MqttController.js

```diff
@@ -0,0 +1,248 @@
+const RobotMqttHandle = require("./handles/RobotMqttHandle");
+const MqttCommonAttributes = require("./MqttCommonAttributes");
+const HomieCommonAttributes = require("./homie/HomieCommonAttributes");
+const mqtt = require("mqtt");
+const asyncMqtt = require("async-mqtt");
+const Logger = require("../Logger");
+
+class MqttController {
+    /**
+     * @param {object} options
+     * @param {import("../core/ValetudoRobot")} options.robot
+     * @param {import("../Configuration")} options.config
+     */
+    constructor(options) {
+        this.loadConfig();
+
+        this.client = null;
+        // Workaround to be able to set two Last Will and Testament payloads, one for Homie and one for HAss
```

Comment:
So it's either Homie or HA?

Reply:
Lol. I realized Homie and Hass want two completely different LWT. So yeah, you need two connections for that.

I temporarily swept the problem under the carpet (~~turbo mode~~) and I'll deal with it better, but I have two solutions in mind:

- Connect to MQTT twice in the same controller; use only one of the connections and use the other one just for the LWT
- Create two instances of the MQTT controller and have one manage homie, the other one hass

I'm more towards the first option, even though it does sound nastier.